### ISSUE TITLE
Why is there a role field in DDB tenant table?

### ISSUE THREAD

**Issue by ilove2git** [aws-samples/aws-saas-factory-bootcamp#61]
when going through lab1 module, I see there's a field name 'role'  whose sample value is tenantAdmin in tenant table. I'm confused about this field. From my point of view, I suppose it should only exist in Cognito User Pool as custom attribute such as tenantAdmin or tenantUser .etc. Please help correct me if wrong. thanks

**Comment by brtrvn** [comment]
You are correct. That is a bug. We do not need to store the authorization role in the tenant manager micro service's data (and the actual tenant registration micro service does not set a "role" attribute when you register using the sample app vs manually invoking the REST endpoint as we do in Lab 1).
